purchase_stock_analytic/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Purchase Stock Analytic",
+    "summary": """
+        Copies the analytic account of the purchase order item to the stock move""",
+    "version": "13.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/account-analytic",
+    "depends": ["purchase", "stock_analytic"],
```

Comment:
@baimont I would say that the exact dependency is 'purchase_stock' as the method you override is defined there.

Reply:
@baimont You can take the last two commits of https://github.com/OCA/account-analytic/pull/410